### ISSUE TITLE
Unable to run

### ISSUE THREAD

**Issue by brendan-pike** [magicgoose/simple_dr_meter#5]
I'm running Python 3.7.3 but when I run I get this;
./main.py 
Traceback (most recent call last):
  File "./main.py", line 12, in <module>
    from audio_io import read_audio_info, read_audio_data, TagKey, TrackInfo, get_tag_with_alternatives
  File "/home/bedlore/Audio-Tools/dr14_t.meter/audio_io.py", line 13, in <module>
    from audio_io.cue.cue_parser import CueCmd, parse_cue_str, read_cue_from_file
ModuleNotFoundError: No module named 'audio_io.cue'; 'audio_io' is not a package

Can you help please

**Comment by brendan-pike** [comment]
Poking around further within this, it seems to rely on https://pypi.org/project/audio.io/#history for which no current version is supported. ie.
$ pip3 install --upgrade audio_io
Collecting audio_io
  Could not find a version that satisfies the requirement audio_io (from versions: )
No matching distribution found for audio_io

Please  advise

**Comment by brendan-pike** [comment]
*bump* would be very grateful if you can find some time to help please.

**Comment by magicgoose** [comment]
sorry, I have currently little time for that,
but audio_io is not a separate package (at least now),
so installing it with pip won't and can't work.

maybe there's a package with a similar name, but it's not what I use here.

try running from this commit, https://github.com/magicgoose/simple_dr_meter/commit/e676d4814801aaaadf85fd4d6787dec2b5a15627
I made some changes that weren't really tested later, and I'm a terrible person so I just pushed to master, but I shouldn't. I just thought that nobody uses it anyway :D
